pull_request_template.md

```diff
@@ -0,0 +1,39 @@
+## Status
+**READY/IN DEVELOPMENT/HOLD**
+
+## Description
+A few sentences describing the overall goals of the pull request's commits.
+
+## Related PRs
```

Comment:
Related PRs could be changed to related tickets and documents so that it's more broad as to what's related.

Reply:
That makes sense as often we have to refer to the support tickets and documentation of the different providers etc.